src/createChannel.js

```diff
@@ -0,0 +1,27 @@
+const { Channel } = require('./channel');
+
+/**
+ * Creates and sets up a channel to AMQP
```

Comment:
```suggestion
 * Creates and sets up a channel to RabbitMQ
```
AMQP is the protocol